ddpui/api/airbyte_api.py

```
@@ -1,6 +1,30 @@
 """Dalgo API for Airbyte"""
 
+# File: ddpui/api/airbyte_api.py
 import os
+
+from ninja import Router, Schema
```

Ruff (0.12.2)

`ninja.Schema` imported but unused

Remove unused import: `ninja.Schema`

(F401)